src/main/java/com/kosherjava/zmanim/util/NOAACalculator.java

```diff
@@ -418,7 +416,7 @@ public static double getSolarAzimuth(Calendar cal, double lat, double lon) {
 		double julianDay = getJulianDay(cal);
 		double julianCenturies = getJulianCenturiesFromJulianDay(julianDay);
 
-		Double eot = getEquationOfTime(julianCenturies);
+		double eot = getEquationOfTime(julianCenturies);
```

Comment:
Not sure why the Double wrapper class was used instead of a regular double. Let me know if this was intentional.